src/MessengerTypes.js

```diff
@@ -0,0 +1,23 @@
+import PropTypes from 'prop-types';
```

Comment:
Pytanie co do typów - większość jest zdefiniowanych tutaj, ale część jest zdefiniowanych w poszczególnych plikach komponentów. Czy stoi za tym jakiś szczególny powód?

Reply:
Z przyzwyczajenia nie trzymam wszystkiego w komponentach, wolę mieć osobne pliki. Zazwyczaj z resztą mam niemalże każdy typ w osobnym pliku. A tutaj z racji tego, że MessageContainerType jest używany w dwóch plikach, to wolałam to mieć wyciągnięte poza komponenty. Te 2 typy, które zostały to pomyłka, chciałam je wyciągnąć do MessegnerTypes.